code.js

```diff
@@ -1,3 +1,56 @@
 function permutationSort(a) {
-    return 0;
+    //Check to see if array is sorted
+    function isSorted(arr) {
+        for (let i = 1; i < arr.length; i++) { 
+            //If a is greater than another. It is not sorted. 
+            if (arr[i-1] > arr[i]) { //ChatGPT changed this part of the code from arr[a] to arr[i-1]
```

Comment:
Not sure if I understand -- there's no variable `a`, so why did you have `arr[a]` to start with?

Reply:
If I am going to be honest, I think I had meant to write arr[i-1], but for some reason, I wrote arr[a], and when I was debugging, I kept skimming over it and never noticed it. I was getting a little impatient because I did not know what the counter example was because it was my first time seeing and I asked ChatGPT for the answer.

Reply:
I updated my readme explaining the mistake I had made.